bin/test/cucumber.rb

```diff
@@ -110,7 +141,7 @@
     exit 0 if failed == 0
 
     # the travis ci environment is not stable enough to have all tests passing
-    exit failed unless Luffa::Environment.travis_ci?
+    exit failed unless RunLoop::Environment.travis? || RunLoop::Environment.azurepipelines?
 
     # we'll take 75% passing as good indicator of health
     expected = 75
```

Comment:
Do we know what the failure rate is on ADO?

Reply:
- Xcode 10.2.1: passrate is 100%
- Xcode 11: we run 6 devices and most often all of them pass successfully.
But sometimes 1 of devices failed so failure rate 1/6 = 16%. I think 75-80% should be ok